Serialization issue with TwoBitFile

```
Caused by: java.io.NotSerializableException: java.nio.HeapByteBuffer
Serialization stack:
        - object not serializable (class: java.nio.HeapByteBuffer, value: java.nio.HeapByteBuffer[pos=0 lim=775499486 cap=775499486])
        - field (class: org.bdgenomics.adam.util.TwoBitFile, name: bytes, type: class java.nio.ByteBuffer)
        - object (class org.bdgenomics.adam.util.TwoBitFile, org.bdgenomics.adam.util.TwoBitFile@249f95a2)
        - field (class: $iwC$$iwC$$iwC$$iwC$$iwC$$iwC$$iwC$$iwC$$iwC$$iwC, name: refFile, type: interface org.bdgenomics.adam.util.ReferenceFile)
```

I believe the fix is marking `bytes` transient.

--- a/adam-core/src/main/scala/org/bdgenomics/adam/util/TwoBitFile.scala
+++ b/adam-core/src/main/scala/org/bdgenomics/adam/util/TwoBitFile.scala
@@ -59,7 +59,7 @@ private object TwoBitFile {
 class TwoBitFile(byteAccess: ByteAccess) extends ReferenceFile {
 
   // load file into memory
-  private[util] val bytes = ByteBuffer.wrap(byteAccess.readFully(0, byteAccess.length().toInt))
+  @transient private[util] val bytes = ByteBuffer.wrap(byteAccess.readFully(0, byteAccess.length().toInt))
   private[util] val numSeq = readHeader()
   // hold current byte position of start of current index record
   var indexRecordStart = TwoBitFile.FILE_INDEX_OFFSET

